templates/dashboard/pricing.html

```diff
@@ -80,104 +80,134 @@
+           target="_blank">
+          Yearly billing - Crypto <br>
+          $30/year
+          <i class="fe fe-external-link"></i>
+        </a>
 
+        <hr>
+        For other payment options, please send us an email at <a href="mailto:hi@simplelogin.io">hi@simplelogin.io</a>.
+        <br>
+        If you have bought a coupon, please go to the <a href="{{ url_for('dashboard.coupon_route') }}">coupon page</a> to
+        apply the coupon code.
+
+      </div>
     </div>
   </div>
 
 
   <script type="text/javascript">
-    Paddle.Setup({vendor: {{ PADDLE_VENDOR_ID }}});
+      Paddle.Setup({vendor: {{ PADDLE_VENDOR_ID }}});
 
-    function upgrade(productId) {
-      bootbox.dialog({
-        title: `Payment with credit card or PayPal via Paddle`,
-        message: `Paddle will ask for an email address for sending out the invoices, please feel free to use an alias. <br>
+      function upgrade(productId) {
+          bootbox.dialog({
+              title: `Payment with credit card or PayPal via Paddle`,
+              message: `Paddle will ask for an email address for sending out the invoices, please feel free to use an alias. <br>
         You don't have to use your SimpleLogin account email address`,
-        size: 'large',
-        onEscape: true,
-        backdrop: true,
-        buttons: {
-          got_it: {
-            label: 'Got it!',
-            className: 'btn-outline-primary',
-            callback: function () {
-              Paddle.Checkout.open({
-                product: productId,
-                success: "{{ success_url }}",
-                passthrough: "{\"user_id\": {{current_user.id}} }"
-              });
-            }
-          },
-        }
-      });
-    }
+              size: 'large',
+              onEscape: true,
+              backdrop: true,
+              buttons: {
+                  got_it: {
+                      label: 'Got it!',
+                      className: 'btn-outline-primary',
+                      callback: function () {
+                          Paddle.Checkout.open({
+                              product: productId,
+                              success: "{{ success_url }}",
+                              passthrough: "{\"user_id\": {{current_user.id}} }"
+                          });
+                      }
+                  },
+              }
+          });
+      }
+
+      function setProtonUpgradeMode(showProton) {
+          document.getElementById("proton-upgrade").style.display = showProton ? "block" : "none";
+          document.getElementById("normal-upgrade").style.display = !showProton ? "block" : "none";
+      }
+
+      function showNormalUpgrade(){
+          document.getElementById("normal-upgrade").style.display = "block";
+          document.getElementById("normal-upgrade-button").style.display = "none";
+      }
+
+      {% if proton_upgrade %}setProtonUpgradeMode(true);{% else %}setProtonUpgradeMode(false);{% endif %}
```

Comment:
show/hide proton upgrade can be done in jinja2 I think by wrapping the `proton-upgrade` with `{% if proton_upgrade %}   {%endif%}`